<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Solar</class>
 <widget class="QMainWindow" name="Solar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1542</width>
    <height>811</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Solar</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QChartView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>30</y>
      <width>550</width>
      <height>550</height>
     </rect>
    </property>
   </widget>
   <widget class="QChartView" name="chart_tiltinso">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>30</y>
      <width>550</width>
      <height>550</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_tilt">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>590</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Draw tilt vs insolation</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>349</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Panel tilt angle</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Solar constant</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDoubleSpinBox" name="dsb_io">
       <property name="suffix">
        <string> W/m2</string>
       </property>
       <property name="maximum">
        <double>999999.989999999990687</double>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="dsb_tilt">
       <property name="suffix">
        <string> Degree</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="pb_di">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>590</y>
      <width>118</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1542</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header>QtCharts</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
